xc7/utils/prjxray_routing_import.py

```diff
@@ -1000,6 +1003,10 @@ def main():
 
     args = parser.parse_args()
 
+    # Initialize the memory log
+    memlog = MemoryLog(args.write_rr_graph + ".memory.log")
```

Comment:
We can get rid of these calls to MemoryLog now

Reply:
Removed.